src/actions/general.js

```diff
@@ -146,8 +146,6 @@ export function setDeviceToken(token) {
     };
 }
 
-EventEmitter.on(General.CONFIG_CHANGED, setServerVersion);
```

Comment:
This never did anything, since calling `setServerVersion` just returned and discarded the thunk. The mobile app handles this event and dispatches on its own accord, and the webapp never used this part of the state tree. I contemplated wiring this up correctly and dispensing with the mobile handling, but deferred for now since not required.